WLC/utils/azure.py

```diff
@@ -0,0 +1,76 @@
+import hashlib
+import logging
+import os
+
+import cv2
+from azure.storage.blob import BlockBlobService, ContentSettings
+
+LOGGER = logging.getLogger()
+
+# Azure spams the logs, this will make it quiet.
+logging.getLogger("azure").setLevel(logging.CRITICAL)
+
+
+def get_block_blob_service():
+    if not os.environ.get('BLOB_ACCOUNT') or not os.environ.get('BLOB_KEY'):
+        raise ValueError('BLOB_ACCOUNT and BLOB_KEY environment variables need to be set.')
+
+    account = os.environ.get('BLOB_ACCOUNT')
+    key = os.environ.get('BLOB_KEY')
+
+    return BlockBlobService(account_name=account, account_key=key)
+
+
+def create_container_not_exists(block_blob_service, container):
+    if not block_blob_service.exists(container):
+        block_blob_service.create_container(container)
+
+
+def save_image_to_azure(container, image):
```

Comment:
Do you think we can set the container name to the current date(YYYYMMDD or something similar)?

Reply:
Based on discussion we decided we wont do this but will use last modified date instead.